catalyst-service/src/main/resources/application.yml

```diff
@@ -317,6 +344,15 @@ initializr:
           - camel
           version: ${camel.version}
           scope: test
+        - name: Camel test Spring
```

Comment:
There is already a camel-test-spring dependency in master. Please merge upstream changes from master in your repo